pyproject.toml

```diff
@@ -39,7 +39,9 @@ license = "BSD-3-Clause"
 readme = "README.md"
 homepage = "https://pycytominer.readthedocs.io/"
 repository = "https://github.com/cytomining/pycytominer"
-include = [{ path = "poetry.lock", format = "sdist" }]
+include = [
+    { path = "poetry.lock", format = "sdist" }, 
+]
```

Comment:
I couldn't tell for sure, but it looked like a formatter might have been used on this file. Would it be worth codifying which formatter and how it should be used?